openrtb2/ad_position.go

```diff
@@ -1,33 +0,0 @@
-package openrtb2
-
-// 5.4 Ad Position
-//
-// Position of the ad as a relative measure of visibility or prominence.
-// This OpenRTB list has values derived from the Inventory Quality Guidelines (IQG).
-// Practitioners should keep in sync with updates to the IQG values as published on IAB.com.
-// Values “4” - “7” apply to apps per the mobile addendum to IQG version 2.1.
-type AdPosition int8
```

Comment:
Note: replaced with https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list--placement-positions-

Reply:
NOTE: completely compatible. Safe to be replaced with adcom1.PlacementPosition.